### DISCUSSION TITLE
How to get clangd to recognize block comments that start mid line and continue to the next line?

### DISCUSSION THREAD

**Discussion by michaelfortunato** [clangd/clangd#1650] [Q&A]
<img width="1977" alt="Screen Shot 2023-05-30 at 7 44 37 PM" src="https://github.com/clangd/clangd/assets/43207799/75f40a8a-7243-4c5a-9f83-6aa9a274b743">
A picture is worth a thousand words^. 
While I know this might not be standard c/cpp syntax, how could I get clangd to recognize block comments that start midline and continue to the next line as valid? 
Thanks!

**Comment by HighCommander4** [comment]
As far as I can tell, what you are showing:

```c++
#define foo /*                                                                 \
        dfsfdj */
#define bar
```

is valid C and C++ syntax, at least clang accepts it without any warnings.

I guess you are asking about the coloring, which in this case is done on the client side, so you'd want to file this in your editor's bug tracker. In vscode it looks fine:

![image](https://github.com/clangd/clangd/assets/1751085/5d9e83be-aa9b-4208-89f2-3cc28de05467)
